src/features/page_action/aggregate/index.js

```diff
@@ -31,17 +32,17 @@ export class Aggregate extends AggregateBase {
 
     register('api-addPageAction', (...args) => this.addPageAction(...args), this.featureName, this.ee)
 
-    this.waitForFlags(['ins']).then(([enabled]) => {
-      if (enabled) {
+    this.waitForFlags(['ins']).then(([insFlag]) => {
+      if (insFlag) {
         const scheduler = new HarvestScheduler('ins', { onFinished: (...args) => this.onHarvestFinished(...args) }, this)
         scheduler.harvest.on('ins', (...args) => this.onHarvestStarted(...args))
         scheduler.startTimer(this.harvestTimeSeconds, 0)
+        this.drain()
       } else {
-        this.blocked = true
+        this.blocked = true // if rum response determines that customer lacks entitlements for spa endpoint, this feature shouldn't harvest
```

Comment:
The insights consumer is tied to the SPA entitlement for really old customer accounts?

Reply:
errant comment, ty.  its only tied to `ins` flag